openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -216,16 +217,40 @@ def _filter_fields(self, field_whitelist, serialized_account):
         return visible_serialized_account
 
 
+class UserAccountDisableHistorySerializer(serializers.ModelSerializer):
+    """
+    Class that serializes User account disable history
+    """
+
+    disabled_by = serializers.SerializerMethodField()
+
+    class Meta(object):
+        model = UserAccountDisableHistory
+        fields = ("created", "comment", "disabled", "disabled_by")
```

Comment:
`disabled_by__username`

Reply:
Is this possible in serializers?

Reply:
Yes yes my dear go ahead. 🎆

Reply:
Confirmed. Not possible at least in the version of Django currently being used by edx-platform